README.markdown

```diff
@@ -95,6 +95,62 @@ end
 
 See `lib/fakefs/spec_helpers.rb` for more info.
 
+**NOTE**: There is a bug currently, that causes FakeFS to fail with the following error:
+```
+TypeError:
+  superclass mismatch for class File
+```
+
+To prevent this, you'll need to require 'pp' *BEFORE* you require 'fakefs/spec_helpers' in your
+spec_helpers file. Like this:
+
+```ruby
+require 'pp'
+require 'fakefs/spec_helpers'
+
+describe "my spec" do
+  include FakeFS::SpecHelpers
+end
+```
+
+Mocking existing files
+----------------------
+
+If you want to load existing files like configuration files which sit with your gem in a
```

Comment:
@grosser It actually does not work. :D I've just created a test in my project, removed all the other crap and just run this test:

```
    it 'correctly setup variables which are loaded from the config file' do
      FakeFS.activate!
      FakeFS::FileSystem.clone(__dir__)
      expect(mineconfig.loglevel).to eq('ERROR')
      expect(mineconfig.upload_path).to eq('/drop')
    end
```

And it faild with loading the config yml file. :) It tries to load it from the cfg folder. But because I'm running it with rake from the root folder, it should have successfully cloned my project.